WMFComponents/Sources/WMFComponents/Components/Article Tabs/WMFArticleTabsView.swift

```diff
@@ -165,8 +165,8 @@ fileprivate struct WMFArticleTabsViewContent: View {
                     .accessibilityHidden(true)
                     .padding(.horizontal, 8)
                     .padding(.top, -8)
+                    .frame(minWidth: 48, minHeight: 48)
                     .contentShape(Rectangle())
-                    .frame(minWidth: 44, minHeight: 44)
```

Comment:
I think this area only updates the main page tab, I see other `44`s in this file.